dev-tools/audius-compose

```diff
@@ -329,12 +395,43 @@ def test_run(protocol_dir, service, args):
             "run",
             "--rm",
```

Comment:
You do `run --rm` anyway, so just use an entrypoint instead to pass args directly?

Reply:
Needed to do this so that `up` works and doesn't run `pytest sleep infinity`